nbs/models.ipynb

```diff
@@ -97,6 +97,7 @@
    ]
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    class AutoNHITS(BaseAuto):
</pre><p>I would highly recommend adding a little comment for each element of the default config and maybe even making a short reference to the paper. It might be even relevant to make a little hint on which parameter we have found to be more important. That way our users can better understand what is happening in the Auto class.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/557/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
I will try to simplify this PR. 
And continue working on the documentation.